homeassistant/components/config/area_registry.py

```diff
@@ -111,6 +113,8 @@ def websocket_delete_area(
         vol.Optional("labels"): [str],
         vol.Optional("name"): str,
         vol.Optional("picture"): vol.Any(str, None),
+        vol.Optional("temperature_entity_id"): vol.Any(str, None),
+        vol.Optional("humidity_entity_id"): vol.Any(str, None),
```

Comment:
All of the parameters in the changed functions are sorted 🔡. We should keep that sorting.

Reply:
done

Reply:
I'd sort the optional items in this dict too since it was sorted before.